src/orange/surf/GeneralQuadric.cc

```diff
@@ -44,6 +44,7 @@ GeneralQuadric::GeneralQuadric(Real3 const& abc,
 /*!
  * Promote from a simple quadric.
  */
+// NOLINTNEXTLINE(bugprone-exception-escape)
```

Comment:
Took me a second to figure this out; but since the GQ constructor it dispatches to *can* throw an exception then perhaps it's better to just drop the `noexcept` from this constructor.

Reply:
Yeah exactly.